Sources/App/Views/Sections/Settings/Server/ServerSettingsViewModel.swift

```diff
@@ -19,7 +19,7 @@ final class ServerSettingsViewModel: ObservableObject {
   /// it will use the observed `workspaceURL` property.
   @Published var workspacePath: String = UserDefaults.standard.url(forKey: UserDefaultKey.workspaceURL)?.path ?? "" {
     didSet {
-      checkURL(workspacePath)
+      workspacePathError = nil
     }
   }
```

Comment:
???????

Reply:
If anything you should delete the `didSet` as a whole

Reply:
I added the doc to explain why the `didSet` is needed